### DISCUSSION TITLE
How to manage permissions when creating files in code server from machine to machine?

### DISCUSSION THREAD

**Discussion by miguelarios** [coder/code-server#2314] [General]
Not so much of an issue with the software, more asking that I use VS Code locally on my Mac AND on my Unraid server using this container. How would it be done to edit files on both machines? Mind you on my Mac I am mounting the directory using SMB with non root while anything i do with code server it is using root.

**Comment by bpmct** [comment]
Just moved this to our discussions forum. Are you able to run the code-server container as a non-root and mount the same directories with SMB and code-server? I believe I was doing this in the past.

**Reply by miguelarios** [reply]
`Are you able to run the code-server container as a non-root` hmm i am seeing if there is a flag in docker run command for me to run as non-root. I'm not sure how to do this.

Since on my Macbook Air I mount the appdata share as user 'miguel' then I need to run my container on my server as a non-root user aka 'miguel' as well. Do you know if there is a flag? I see this 
![image](https://user-images.githubusercontent.com/24581058/100017260-3c9ef600-2da0-11eb-95d8-f13a887c7ba0.png)
